vendor/assets/stylesheets/normalize.css

```diff
@@ -0,0 +1,419 @@
+/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
I'm new here (they ruby world) so i'm not sure where best practices but when we include third-party in our repo we should probably track its origin with README so we can track where it comes from and how to update it.